### PR TITLE
docs(graphql): Update IN filter documentation

### PR DESCRIPTION
**Pull request by bucanero**
Fixes DOC-192

### CODE REVIEW

File: content/graphql/queries/search-filtering.md [hunk 1]
```diff
@@ -222,8 +222,9 @@ queryStudent(filter: {name: between: {min: "ba", max: "hz"}}){
 
 You can filter query results to find objects with one or more specified values using the
 `in` keyword. This keyword can find matches for fields with the `@id` directive
-applied. This filter is also supported on `string` and `enum` types with a
+applied. This filter is supported on `string` and `enum` types with a
```
Comment (minhaj-shakeel) [inline content/graphql/queries/search-filtering.md:225]: Remove this line "This filter is supported .......". Incorporate `string` and `enum` types with other types.

File: content/graphql/queries/search-filtering.md [hunk 2]
```diff
@@ -222,8 +222,9 @@ queryStudent(filter: {name: between: {min: "ba", max: "hz"}}){
...
 [`Hash` or `Exact` index](/graphql/schema/search/#string-exact-and-hash-search).
+The `in` filter is also supported for all scalar data types such as `Int`, `Int64`, `Float`, and `DateTime`.
```
Comment (minhaj-shakeel) [inline content/graphql/queries/search-filtering.md:227]: Incorporate `string` and `enum` types in this line only.

File: content/graphql/queries/search-filtering.md (hunk 1 above)
Comment (bucanero) [reply content/graphql/queries/search-filtering.md:225]: fixed

File: content/graphql/queries/search-filtering.md (hunk 2 above)
Comment (bucanero) [reply content/graphql/queries/search-filtering.md:227]: fixed

**Pull request merged**